### DISCUSSION TITLE
Analytics script 404 not found error

### DISCUSSION THREAD

**Discussion by vkotu** [vercel/community#1308] [Help] [answered]
### Summary

I have added the Analytics into my project as per https://vercel.com/docs/concepts/analytics/audiences/quickstart

But im seeing 404 error in console and network tab.
<img width="861" alt="image" src="https://user-images.githubusercontent.com/8981254/213626648-81396a32-e362-4acb-ae74-acb740c874d8.png">

### Additional information

My code snippet
![image](https://user-images.githubusercontent.com/8981254/213623200-bdac281c-e766-4590-85e8-52abda1021f6.png)

**Comment by swarnava** [comment, marked as answer]
Looks like Analytics isn't enabled in your Project. You need to enable it first : https://vercel.com/vkotu/portfolio-next/analytics?device=desktop&timePeriod=1d&percentile=75&timeInterval=3h&sort=datapoints&search=

**Reply by vkotu** [reply]
I did enabled the Audiences .
![8A1247AC-2B4D-4326-88A0-43FBA25F5743](https://user-images.githubusercontent.com/8981254/213637048-319ce7bf-e391-499d-ba37-482f26ffc820.jpeg)

**Reply by vkotu** [reply]
Ok I can see now. Thanks for help
